scripts/nlp_language_modeling/hf_t5-v1_1_to_nemo.py

```diff
@@ -0,0 +1,383 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This script generates a NeMo-Megatron compatible `.nemo` file for a Huggingface T5-v1_1 model.
+
+List of Huggingface models that this script can covert:
+
+1. google/t5-v1_1-small
+2. google/t5-v1_1-base
+3. google/t5-v1_1-large
+4. google/t5-v1_1-xl
+5. google/t5-v1_1-xxl
+6. google/mt5-small
+7. google/mt5-base
+8. google/mt5-large
+9. google/mt5-xl
+10. google/mt5-xxl
+11. google/ul2
+13. bigscience/T0pp
+14. google/t5-small-lm-adapt
+15. google/t5-base-lm-adapt
+16. google/t5-large-lm-adapt
+17. google/t5-xl-lm-adapt
+18. google/t5-xxl-lm-adapt
+
+Use instructions:
+
+python hf_t5_to_nemo_coverter.py \
```

Comment:
name of script should be updated to `hf_t5-v1_1_to_nemo.py` in this doc string.

Reply:
Good catch.